e2e/helpers/common.helper.ts

```diff
@@ -1,14 +1,17 @@
 import { Payment, Transaction } from 'bitcoinjs-lib';
 import { IndexerService } from '@/indexer/indexer.service';
+import {
+    Transaction as EntityTransaction,
+    TransactionOutput,
+} from '@/transactions/transaction.entity';
 import { SATS_PER_BTC } from '@/common/constants';
 import * as currency from 'currency.js';
-import { varIntSize } from '@/common/common';
 
 export function generateScanTweak(
```

Comment:
you should also update the function name to capture the change in return type, so change it to something like `generateScanTweakAndOutputEntity`